app/src/main/java/br/com/dionataferraz/vendas/activities/TransactionRegistryActivity.kt

```diff
@@ -46,4 +48,15 @@ class TransactionRegistry : AppCompatActivity() {
             Toast.LENGTH_SHORT
         ).show()
     }
+
+    private fun getCheckedType(): TransactionType {
+        if (binding.rdGasStation.isChecked) {
+            return TransactionType.GAS_STATION
+        } else if (binding.rdMarket.isChecked) {
+            return TransactionType.MARKET
+        } else if (binding.rdPub.isChecked) {
+            return TransactionType.PUB
+        }
+        return TransactionType.UNKNOWN
```

Comment:
Podemos fazer dessa forma para melhorar a legibilidade do código
```suggestion
        return when{
            binding.rdGasStation.isChecked -> TransactionType.GAS_STATION
            binding.rdMarket.isChecked -> TransactionType.MARKET
            binding.rdPub.isChecked -> TransactionType.PUB
            else -> TransactionType.UNKNOWN
        }
```